lib/utils/hardware.dart

```diff
@@ -4,15 +4,14 @@ import 'dart:math';
 import 'dart:typed_data';
 
 import 'package:ffi/ffi.dart';
-import 'package:loggy/loggy.dart';
 import 'package:momento_booth/exceptions/win32_exception.dart';
+import 'package:momento_booth/main.dart';
 import 'package:momento_booth/managers/settings_manager.dart';
+import 'package:momento_booth/utils/logger.dart';
 import 'package:pdf/pdf.dart';
 import 'package:pdf/widgets.dart' as pw;
 import 'package:win32/win32.dart';
 
-final _loggy = Loggy<UiLoggy>("hardware utils");
```

Comment:
Not the most important, but is there a way to still say where the log came from?

Reply:
Not officially, but I'm currently having the `Logger` mixin formats messages using this:

`"[$runtimeType] $msg"`